exporter/prometheusexporter/collector.go

```diff
@@ -162,13 +162,103 @@ func (c *collector) convertMetric(metric pmetric.Metric, resourceAttrs pcommon.M
 		return c.convertSum(metric, resourceAttrs, scopeName, scopeVersion, scopeSchemaURL, scopeAttributes)
 	case pmetric.MetricTypeHistogram:
 		return c.convertDoubleHistogram(metric, resourceAttrs, scopeName, scopeVersion, scopeSchemaURL, scopeAttributes)
+	case pmetric.MetricTypeExponentialHistogram:
+		return c.convertExponentialHistogram(metric, resourceAttrs, scopeName, scopeVersion, scopeSchemaURL, scopeAttributes)
 	case pmetric.MetricTypeSummary:
 		return c.convertSummary(metric, resourceAttrs, scopeName, scopeVersion, scopeSchemaURL, scopeAttributes)
 	}
 
 	return nil, errUnknownMetricType
 }
 
+// defaultZeroThreshold matches the remote-write translator's default for native histograms
+// when an explicit zero threshold is not provided in the datapoint.
+const (
+	defaultZeroThreshold = 1e-128
+	CBNHScale            = -53
```

Comment:
can you make this lower-cased so it isn't part of the public API of the package?